### PR TITLE
added index html

### PR DESCRIPTION
**Pull request by carys23**

### CODE REVIEW

File: cage/cryptids/templates/index.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{% extends "base.html" %}
+{% load static %}
+
+
+{% block title %}Cryptids{% endblock %}
+{% block content %}
+
+
+
```
Comment (tamjankowska) [inline cage/cryptids/templates/index.html:9]: You've got some blank space here you could delete maybe?

File: cage/cryptids/templates/index.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+<ul>
+        {% for animals in animalss %}
```
Comment (tamjankowska) [inline cage/cryptids/templates/index.html:12]: For your `for loop`, you could just do `for animal in animals`, reads a bit more easily :)

File: cage/cryptids/templates/index.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        <li>
+                <a href="{% url 'cryptids:detail' animals.id %}">
```
Comment (tamjankowska) [inline cage/cryptids/templates/index.html:14]: Does `cryptids:detail` work here? Is that what you're calling your view?

**Review by tamjankowska — CHANGES_REQUESTED** [review]
Getting there, just a few little things :)

File: cage/cryptids/templates/index.html (hunk 2 above)
Comment (carys23) [reply cage/cryptids/templates/index.html:12]: Yes I was a little confused on this what it should be thanks

**Comment by carys23** [comment]
wrong

**Comment by carys23** [comment]
wrong

**Pull request closed without merging**